tls/s2n_prf.c

```diff
@@ -607,8 +607,7 @@ int s2n_prf_key_expansion(struct s2n_connection *conn)
 
     label.data = key_expansion_label;
```

Comment:
should this also be `blob_init(...)`

Reply:
There are a lot of blobs that have been created without using the s2n_blob_init function. The reason why I initialized the out blob in this file is because otherwise the stuffer_init function will fail. Fixing blobs that have been initialized incorrectly is beyond the scope of this pr.

Reply:
OK.  Lets make an issue to track that tho.